xml/schema/types/oblocks-2-9-6.xsd

```diff
@@ -43,9 +43,9 @@
         <xs:element name="oblock" minOccurs="0" maxOccurs="unbounded">
           <xs:complexType>
             <xs:sequence>
```

Comment:
I think it would be better to change line 45 from xs:sequence to xs:all since xs:sequence dictates a specific order of child elements while xs:all does not.